project/Dependencies.scala

```diff
@@ -33,6 +33,9 @@ object Dependencies {
     lazy val agreementProcessClient    = namespace %% "interop-be-agreement-process-client" % agreementProcessVersion
     lazy val agreementManagementClient =
       namespace %% "interop-be-agreement-management-client" % agreementManagementVersion
+    lazy val agreementManagementModels =
+      namespace %% "interop-be-agreement-management-models" % agreementManagementVersion
```

Comment:
this could be avoided if we hardcode filters as strings, but I preferred to keep the compile time check, even if thin.
Let me know